lib/modules/manager/types.ts

```diff
@@ -198,9 +198,9 @@ export interface UpdateArtifactsResult {
   file?: FileChange;
 }
 
-export interface UpdateArtifact {
+export interface UpdateArtifact<T = Record<string, any>> {
```

Comment:
wouldn't like to see unknown instead of any in future

Reply:
I mainly chose `any` as it's currently the default type for `PackageDependency`'s generic type. I can change it here already if you'd like?

Reply:
if it doesn't cause too many issues 🤗